config/local.env.example

```diff
@@ -1,3 +0,0 @@
-POWERPLAY_JWT_SECRET=your_secret_here
```

Comment:
Is there a reason you deleted this?

Reply:
I didnt know if we were supposed to have the example so I just deleted it

Reply:
Yeah Skyler added that, please keep it